awx_collection/tests/integration/targets/tower_project_update/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Update a project without waiting
+  tower_project_update:
+    name: "AWX-Collection-tests-tower_job_template-proj-{{ test_id }}"
```

Comment:
The object name structure is also: `AWX-Collection-tests-<module>-<asset type>-<id>`. So tower_job_template should be later to tower_project_update. And, since you use it down below it should probably be in a variable. Check out tower_export for an example of creating a test_id, creating multiple assets and then running things in a block to try and ensure cleanup on a failed test.